### ISSUE TITLE
Headless Zombies 2 - Character not displayed

### ISSUE THREAD

**Issue by desertking** [ruffle-rs/ruffle#10327]
### Describe the bug

https://www.newgrounds.com/portal/view/651073?emulate=flash - Hey this one is missing the character when you want to play

### Expected behavior

Show him and run :)

### Affected platform

Online demo

### Operating system

All

### Browser

All

**Comment by n0samu** [comment]
The problem is related to this warning:
`WARN ruffle_core::stub: Encountered stub: AVM2 flash.utils.Dictionary constructor with weak keys`

With #10069, the character enters the stage, but quickly disappears due to these two errors:

```
ERROR ruffle_core::avm2::activation: TypeError: Error #1009: Cannot access a property or method of a null object reference. (accessing field: addEventListener)
        at fl.core::UIComponent/callLaterDispatcher()
        at flash.display::DisplayObject/flash.display::DisplayObject()
        at flash.display::InteractiveObject/flash.display::InteractiveObject()
        at flash.display::DisplayObjectContainer/flash.display::DisplayObjectContainer()
        at flash.display::Sprite()
        at flash.display::MovieClip()
        at resources.screens::BaseLevelMenuScreen()
        at app.view.screens::LevelMenuScreen()
        at app.view.screens::StoryScreen/skipBtnClickHandler()
```
```
ERROR ruffle_core::avm2::activation: TypeError: Error #1009: Cannot access a property or method of a null object reference. (accessing field: addEventListener)
        at fl.core::UIComponent/callLaterDispatcher()
        at flash.display::DisplayObject/flash.display::DisplayObject()
        at flash.display::InteractiveObject/flash.display::InteractiveObject()
        at flash.display::DisplayObjectContainer/flash.display::DisplayObjectContainer()
        at flash.display::Sprite()
        at flash.display::MovieClip()
        at resources.screens::BaseGameScreen()
        at app.view.screens::GameScreen()
        at app.view.screens::LevelMenuScreen/levelBtnClickHandler()
```

**Comment by n0samu** [comment]
Fixed by #10864. (This turned out not to be related to dictionary weak keys at all!)
